ntbs-service/Services/UserService.cs

```diff
@@ -122,7 +123,7 @@ public async Task<string> GetUserDisplayName(ClaimsPrincipal user)
             if(displayName.Contains("@")){
```

Comment:
In the case where the name isn't an email, don't we also want to format the name from the identity?

Reply:
Yes I agree, added in 3ddca0f4